linux/Jenkinsfile

```diff
@@ -174,18 +175,67 @@ def CheckAndUpload(String Target, String Distro, String BuildArch, String Versio
       echo "Target Doesnt Exist - Upload Files"
       switch(Distro) {
            case "Debian":
-               jf 'rt u **/build/ospackage/temurin-*${BUILDARCH}.deb deb/pool/main/t/temurin-${VERSION}/ --target-props=${DISTROLIST}deb.component=main;deb.architecture=${BUILDARCH} --flat=true'
-               break
+               // Not RISC Use JF CLI
+               if (BuildArch != "riscv64") {
+                 jf 'rt u **/build/ospackage/temurin-*${BUILDARCH}.deb deb/pool/main/t/temurin-${VERSION}/ --target-props=${DISTROLIST}deb.component=main;deb.architecture=${BUILDARCH} --flat=true'
+               }
+               if (BuildArch == "riscv64") {
+                     // RISC Use RT upload
+                     node("dockerBuild&&linux&&x64") {
```

Comment:
Can we extract this duplicated code somehow?

Reply:
Yup, im just testing a change to improve this.